### PR TITLE
Add subset_sizes to FSelectorRFE

### PR DESCRIPTION
**Pull request by be-marc**
Requested in #1

### CODE REVIEW

File: R/FSelectorRFE.R [hunk 1]
```diff
@@ -96,21 +101,19 @@ FSelectorRFE = R6Class("FSelectorRFE",
             self$importance = importance_average(learners, feature_names)
           }
           # Eliminate the most unimportant features
-          states = as.list(!feature_names %in% names(
-            self$importance[1:archive$n_batch]))
+          states = as.list(feature_names %in% names(self$importance[seq(i)]))
```
Comment (mllg) [inline R/FSelectorRFE.R:104]: Can `seq(i)` go out of bounds?

Comment (be-marc) [reply R/FSelectorRFE.R:104]: We assert that `pars$subset_sizes` is in decreasing order, ` < length(feature_names)` and `> 1`. So it should not go out of bounds.

Comment (be-marc) [reply R/FSelectorRFE.R:104]: And `self$importance` is always a vector of length `feature_names`

**Comment by be-marc** [comment]
Implemented fraction of features to retain and number of features to remove in each iteration. Default is `feature_fraction = 0.5` and `min_features = 1`.

**Pull request merged**